src/lib/nvme_pt_ioctl.c

```diff
@@ -40,6 +40,9 @@
 
 #define MIN_IO_BUFFER_LEN 2048
 
+#define OPAL_BLOCKSID_COMID  5
+#define BLOCKSID_PAYLOAD_SZ  512
```

Comment:
is there a particular reason for this size? at minimum we support buffer 2048 B in size

Reply:
I don't see the blockSID spec mandate that, however TWIDL is sending 512 byte to drive regardless the supported buffer size is.

Reply:
please take a look at architecture core specifiocation chapter 5.2.2.4 "Communications minimums"

Reply:
see my other comment addressing your same concern.